src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  width: 96px;
+  height: auto;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: 80%;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
Use the whole `stars` block from the previous task. It should be possible to set any amount of stars